src/cuga/backend/server/manage_routes.py

```
@@ -342,6 +402,81 @@ async def _load_and_patch_draft(agent_id: str, section: str, value: Any) -> dict
+                except Exception as cerr:
+                    logger.warning(f"copy {source} -> {target} failed: {cerr}")
+                    return {"triggered": False, "target": target, "error": "copy_failed"}
+        finally:
+            live_engine._reindex_in_progress.discard(source)
+            live_engine._reindex_deferred.discard(source)
+
+    try:
+        r = await live_engine.reindex(target)
+        triggered.append({"collection": target, "result": r})
+        ok = bool(r and r.get("status") not in (None, "no_documents"))
+    except Exception as rerr:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)